commit 914245808bd77ef9f0f5403957de3c10ba826904
Author: Lukas Wunner <lukas@wunner.de>
Date:   Sat Jan 19 08:06:48 2019 +0100

    bcm2835-mmc: Fix struct mmc_host leak on probe
    
    The BCM2835 MMC host driver requests the bus address of the host's
    register map on probe.  If that fails, the driver leaks the struct
    mmc_host allocated earlier.
    
    Fix it.
    
    Signed-off-by: Lukas Wunner <lukas@wunner.de>
    Cc: Frank Pavlic <f.pavlic@kunbus.de>

diff --git a/drivers/mmc/host/bcm2835-mmc.c b/drivers/mmc/host/bcm2835-mmc.c
index fd2225a388c4..c4ca979995f9 100644
--- a/drivers/mmc/host/bcm2835-mmc.c
+++ b/drivers/mmc/host/bcm2835-mmc.c
@@ -1439,7 +1439,8 @@ static int bcm2835_mmc_probe(struct platform_device *pdev)
 	addr = of_get_address(node, 0, NULL, NULL);
 	if (!addr) {
 		dev_err(dev, "could not get DMA-register address\n");
-		return -ENODEV;
+		ret = -ENODEV;
+		goto err;
 	}
 	host->bus_addr = be32_to_cpup(addr);
 	pr_debug(" - ioaddr %lx, iomem->start %lx, bus_addr %lx\n",
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1388_914245808bd77ef9f0f5403957de3c10ba826904.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1388_914245808bd77ef9f0f5403957de3c10ba826904.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
